.github/CODEOWNERS

```diff
@@ -28,3 +28,5 @@ yarn.lock                    @janus-idp/maintainers-plugins
 /plugins/rbac-backend/       @janus-idp/maintainers-plugins @gorkem @AndrienkoAleksandr @PatAKnight
 /plugins/rbac-common/        @janus-idp/maintainers-plugins @gorkem @AndrienkoAleksandr @PatAKnight
 /plugins/notifications-frontend  @janus-idp/maintainers-plugins @mareklibra
+/plugins/feedback           @janus-idp/maintainers-plugins @yashoswalyo @riginoommen @deshmukhmayur 
+/plugins/feedback-backend   @janus-idp/maintainers-plugins @yashoswalyo @riginoommen @deshmukhmayur
```

Comment:
codeowners file show error

@kadel / @tumido do you have an idea, can you help with this?

Reply:
I think that this is ok, we can ignore those warnings.

I'm working on onboarding this repo to Openshift-ci merge automatization (Prow). Once this is done I'll migrate Github CODEOWNERS to [Prow OWNERS](https://www.kubernetes.dev/docs/guide/owners/) file.